aksel.nav.no/website/components/website-modules/search/providers/SearchNavigationProvider.tsx

```diff
@@ -28,7 +28,7 @@ export const SearchNavigationProvider = ({
 
   /* Add a small delay to get a precieved smoother navigation */
   useEffect(() => {
-    let timeout: NodeJS.Timeout;
+    let timeout: ReturnType<typeof setTimeout>;
```

Comment:
Why change this?

Reply:
Since `setTimeout` is implemented by the js runtime (engine implements it, but runtime exposes the function), it could in theory differ between nodejs, deno, hermes etc. So `ReturnType<typeof setTimeout>` will be more correct in this scenario.

Found this since `let timeout;` caused errors elsewhere and thus lead me to this instance.